Makefile

```diff
@@ -2,6 +2,20 @@ LOG_LEVEL ?= spin=trace
 CERT_NAME ?= local
 SPIN_DOC_NAME ?= new-doc.md
 
+## dependencies for e2e-tests
+E2E_TESTS_DOCKERFILE ?= e2e-tests.Dockerfile
+MYSQL_IMAGE ?= mysql:8.0.22
+REDIS_IMAGE ?= redis:7.0.8-alpine3.17
+POSTGRES_IMAGE ?= postgres:14.7-alpine
+
+## overrides for aarch64
+ifneq ($($(shell uname -p)),x86_64)
+	MYSQL_IMAGE=arm64v8/mysql:8.0.32
```

Comment:
Perhaps these can also use `?=` such that they can be overridden (to test a different version during development, etc)?

Reply:
as all these env are initialized above using `?=`, with in this if condition we cannot use`?=`. Also, because we already have `?=` before this if condition, user can still override them.